cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,65 @@
 /// <reference types='cypress' />
 
+const { generateUser } = require('../support/generate');
+
 describe('Student Registration page', () => {
-  before(() => {
-    
-  });
 
-  it('', () => {
+  const {firstName, lastName, email, gender, mobileNumber, birth, subject, hobby, address} = generateUser();
+
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/automation-practice-form')
```

Comment:
as long as you use config file u can use base url

Reply:
check your code for missing semicolons

Reply:
added